go.mod

```diff
@@ -0,0 +1,87 @@
+module k8s.io/cloud-provider-openstack
+
+go 1.13
+
+require (
+	github.com/NYTimes/gziphandler v1.1.1 // indirect
```

Comment:
Is this file automatically generated or any manual changes of versions done?

Reply:
Ok, could you clarify one thing (sorry , still getting hang on using gomod), how the master branch things taken care, like we need to update from time to time , running go mod tidy or something ? 
And one more thing, doesnt vendor directory need to created again? because previous vendor was based on dep

Reply:
With go modules go automatically downloads all the dependencies into the go system cache, there is no need to maintain the `vendor` directory. `go mod vendor` makes sense only for projects, which commit the `vendor` dir inside the repo.

Reply:
If you want to update dependencies, then you usually set the new dependency version in the `go.mod` and run `go mod download`. It is also possible to run `go mod download %dep_url%@version`, but sometimes it doesn't work. Then you run `go mod tidy`